examples/linux/policyfile_cookbook/.gitignore

```diff
@@ -0,0 +1,22 @@
+.vagrant
```

Comment:
Do not include a gitignore in subdirectories

Reply:
See comment below about this being an example cookbook (though, I see why this concerns you).

Reply:
I see your point here and retract my comment for now